source/unified-test-format/tests/Makefile

```diff
@@ -1,48 +1,54 @@
 SCHEMA=../schema-1.23.json
 
-.PHONY: all invalid valid-fail valid-pass atlas-data-lake versioned-api load-balancers gridfs transactions transactions-convenient-api crud collection-management read-write-concern retryable-reads retryable-writes sessions command-logging-and-monitoring client-side-operations-timeout HAS_AJV
+.PHONY: all atlas-data-lake auth change-streams client-side-encryption client-side-operations-timeout collection-management command-logging-and-monitoring command-logging-and-monitoring/logging command-logging-and-monitoring/monitoring connection-monitoring-and-pooling connection-monitoring-and-pooling/logging crud gridfs index-management load-balancers read-write-concern retryable-reads retryable-writes run-command server-discovery-and-monitoring server-selection server-selection/logging sessions transactions-convenient-api transactions unified-test-format unified-test-format/invalid unified-test-format/valid-fail unified-test-format/valid-pass versioned-api HAS_AJV
 
-all: invalid valid-fail valid-pass atlas-data-lake versioned-api load-balancers gridfs transactions transactions-convenient-api change-streams crud collection-management read-write-concern retryable-reads retryable-writes sessions command-logging-and-monitoring client-side-operations-timeout client-side-encryption
+all: atlas-data-lake auth change-streams client-side-encryption client-side-operations-timeout collection-management command-logging-and-monitoring connection-monitoring-and-pooling crud gridfs index-management load-balancers read-write-concern retryable-reads retryable-writes run-command server-discovery-and-monitoring server-selection sessions transactions-convenient-api transactions unified-test-format versioned-api
 
-invalid: HAS_AJV
-	@# Redirect stdout to hide expected validation errors
-	@ajv test -s $(SCHEMA) -d "invalid/*.yml" --invalid > /dev/null && echo "invalid/*.yml passed test"
+# Keep specifications sorted alphabetically
+# When adding a new specification, remember to add it to the all and .PHONY targets above
+# For specifications that contain
```

Comment:
> For specifications that contain

Was information lost?

Reply:
Yes, I seemed to have stopped writing mid-sentence. Fixed now.